primo/data_parser/well_data.py

```diff
@@ -864,12 +872,76 @@ def _process_input_data(self):
 
         LOGGER.info("Completed processing the essential inputs.")
 
+    def _get_state(self) -> str:
```

Comment:
Since the users will be interacting with the WellData class, I was wondering if we should only keep the methods that might be useful for the users. The rest of the methods can be moved to utilities. I was planning to restructure the code that way later on. If you are okay with it, shall we move these methods to census_utils.py file (The input to these functions must be a WellData object)?

Reply:
sure, happy to